docs/datadog_dashboard.md

```diff
@@ -0,0 +1,138 @@
+# Datadog Dashboards
+This feature is in preview.
+
+## Overview
+The DatadogDashboard Custom Resource Definition (CRD) allows users to create [dashboards][1] using the Operator and manage them as Kubernetes resources.
+
+## Prerequisites
+
+- Datadog Operator v1.9+
+- **[Helm][2]** for deploying the Datadog Operator
+- **[`kubectl` CLI][3]** for installing a `DatadogDashboard`
+
+Tests were performed on Kubernetes versions >= `v1.30.0`
```

Comment:
Is this line necessary? It seems to me like this should either be a prereq or shouldn't be mentioned.